testing/trino-product-tests-launcher/src/main/java/io/trino/tests/product/launcher/env/environment/SinglenodeHdp3.java

```diff
@@ -58,6 +60,21 @@ public void extendEnvironment(Environment.Builder builder)
             dockerContainer.withCopyFileToContainer(
                     forHostPath(dockerFiles.getDockerFilesHostPath("conf/environment/singlenode-hdp3/apply-hdp3-config.sh")),
                     CONTAINER_HADOOP_INIT_D + "apply-hdp3-config.sh");
+            dockerContainer.withCopyFileToContainer(
+                    forHostPath(dockerFiles.getDockerFilesHostPath("conf/environment/singlenode-hdp3/json-serde-1.3.9-SNAPSHOT-jar-with-dependencies.jar")),
+                    "/docker/openx-serde.jar");
```

Comment:
would it be possible to get openx as maven dependency, e.g:
```
            <plugin>
                <groupId>org.apache.maven.plugins</groupId>
                <artifactId>maven-dependency-plugin</artifactId>
                <executions>
                    <execution>
                        <id>copy</id>
                        <phase>package</phase>
                        <goals>
                            <goal>copy</goal>
                        </goals>
                        <configuration>
                            <skip>false</skip>
                            <artifactItems>
                                <artifactItem>
                                    <groupId>io.trino</groupId>
                                    <artifactId>trino-cli</artifactId>
                                    <version>${dep.trino.version}</version>
                                    <classifier>executable</classifier>
                                    <type>jar</type>
                                    <outputDirectory>${project.build.directory}</outputDirectory>
                                </artifactItem>
```
in product tests launcher xml

Reply:
Sure, I can do that, but what is the intention of adding it as a product tests launcher dependency? Would this impact the classpath inside Presto or Hadoop dockers? The test launcher doesn't read the data with the serde itself.

Reply:
The intention is not to store binary files with repo and rely on dependencies instead.

Reply:
Oh, sure, I will have to upload this to a Maven repository somewhere. No problem.